tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -846,7 +847,9 @@ func isWebAction(client *whisk.Client, qname QualifiedName) bool {
     if err != nil {
         whisk.Debug(whisk.DbgError, "client.Actions.Get(%s) error: %s\n", fullActionName, err)
         whisk.Debug(whisk.DbgError, "Unable to obtain action '%s' for web action validation\n", fullActionName)
+        errMsg = wski18n.T("API action does not exist")
     } else {
+        errMsg = wski18n.T("API action is not a web-action")
         weVal := getValue(action.Annotations, "web-export")
         if (reflect.TypeOf(weVal) == nil) {
```

Comment:
when originally coded, there was a compiler error calling out a mismatch in type comparison, thus the use of `reflect`; however, I recently tried the same `nil` comparison, and it's working fine.  I moved from go 1.7 to 1.8 recently.. although I'm not entirely sure that's the cause of the earlier compilation failure...